packages/utilities/src/utilities/sameheight/sameheight.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * Utility that sets elements under a same parent to the sameheight.
+ *
+ * It uses the selector passed to the `sameheight` attribute, to identify
+ * which items should be aligned
+ *
+ * @example
+ * `HTML`
+ *    <div sameheight=".adjustme">
+ *      <elemA class="adjustme">
+ *      <elemB class="adjustme">
+ *    </div>
+ *
+ * `JS`
+ * import {sameheight} from '@carbon/ibmdotcom-utilities';
+ *
+ * sameheight();
+ */
+function sameheight() {
+  /**
+   * Internal function made for avoiding adding more eventlisteners
+   */
+  function setHeight() {
+    if (window.innerWidth > 671) {
```

Comment:
we could have an optional field that sets when this logic will start to kick off (s, m, lg, etc), and default logic is that it will always be on regardless of breakpoint.

Reply:
I guess this approach makes sense, so we can have the logic as a second parameter, I'll apply this to the current logic